README.md

```diff
@@ -5,6 +5,7 @@
 [![License: GPL v3](https://img.shields.io/badge/License-GPLv3-blue.svg)](https://www.gnu.org/licenses/gpl-3.0)
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/omicsNLP/Auto-CORPus/main.svg)](https://results.pre-commit.ci/latest/github/omicsNLP/Auto-CORPus/main)
 [![PyPI version](https://badge.fury.io/py/autocorpus.svg)](https://badge.fury.io/py/autocorpus)
+[![codecov](https://codecov.io/gh/ImperialCollegeLondon/Auto-CORPus/graph/badge.svg?token=6WWASKCH66)](https://codecov.io/gh/ImperialCollegeLondon/Auto-CORPus)
```

Comment:
This is the wrong repo. I'm guessing the token value will also be different

```suggestion
[![codecov](https://codecov.io/gh/omicsNLP/Auto-CORPus/graph/badge.svg?token=6WWASKCH66)](https://codecov.io/gh/omicsNLP/Auto-CORPus)
```

Reply:
Oh, crap. I *was* looking at the settings for the wrong repo... which also explains why I couldn't activate the app :laughing: